@shared/api/internal.ts

```diff
@@ -605,18 +605,23 @@ export const getTokenPrices = async (tokens: string[]) => {
 };
 
 export const getDiscoverData = async () => {
-  const url = new URL(`${INDEXER_URL}/protocols`);
+  const url = new URL(
+    "https://freighter-backend-v2-stg.stellar.org/api/v1/protocols",
+  );
   const response = await fetch(url.href);
   const parsedResponse = (await response.json()) as {
     data: {
-      description: string;
-      icon_url: string;
-      name: string;
-      website_url: string;
-      tags: string[];
-      is_blacklisted: boolean;
-    }[];
+      protocols: {
```

Comment:
it looks like the API payload changed slightly since this was merged. Updating here